### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by 4mierS**
# List of changes
## Changes
* Updated Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,54 @@
...
+</p>
+
+
+
+
+## Contact
+
+SWE|B4        - [Discord](https://discord.gg/dz8ZvnhqFT)
+
+SWE Gruppe B4 - [Telegram](https://t.me/+KiS5pwEKKkxmZTNi)
```
Comment (cr4yfish) [inline README.md:52]: Remove Telegram link.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,54 @@
...
+<img src="https://user-images.githubusercontent.com/85135956/212556218-8a0a9523-a6a6-4346-92be-3d6a6c8e2ea0.png" alt="drawing" width="200"/>
+<img src="https://user-images.githubusercontent.com/85135956/212556235-d031d9fc-64a8-48f4-9897-8ad6bf22c981.png" width="200"/>
+</p>
+
+
+
+
+## Contact
+
+SWE|B4        - [Discord](https://discord.gg/dz8ZvnhqFT)
```
Comment (cr4yfish) [inline README.md:50]: Remove Discord Link.

File: README.md [hunk 3]
```diff
@@ -1,17 +1,54 @@
...
+*  [PyramidQuest.rar](https://github.com/B4-Group/swe_b4/releases/download/v1.3/PyramidQuest.v1.3.rar)
+1.  Open the folder PyrmaidQuest
+2.  Click on PyramidQuest
+3.  Click on `PyramidQuest.rar`
+
+The Game get started
+
+## Feature 
+
+See the [Lastenheft](https://drive.google.com/file/d/1Zo8HbJB4aVkVHRcmlVdS-qxpoCjMNVfY/view?usp=sharing) for a full list of proposed features.
```
Comment (cr4yfish) [inline README.md:32]: This Link won't work for anyone other than out team members.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,54 @@
...
+To get a local copy up and running follow these simple steps.
 
-### Qualitätmanagement:
-* Nam
-* Ikram
+### Prerequisites
+
+You need a Windows 10 operating system or newer.
+
+### Installation
```
Comment (cr4yfish) [inline README.md:16]: Installation part needs Development instructions.

Installation instructions need to be from "visiting github" all the way to "execute PyramidQuest.exe"

File: README.md [hunk 5]
```diff
@@ -1,17 +1,54 @@
...
-### Vertriebsmanagement:
-* Amier
-* Lucas
+* [Unity]
 
-### Techmanagement:
-* Baran
-* Hannah
+## Getting Started
+To get a local copy up and running follow these simple steps.
```
Comment (cr4yfish) [inline README.md:10]: This line can be removed.

**Review by cr4yfish — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (4mierS) [reply README.md:32]: public?

**Review by cr4yfish — APPROVED** [review]

**Pull request merged**